libraries/holochain-service-client/src/main/java/org/holochain/androidserviceruntime/holochain_service_client/HolochainServiceClient.kt

```diff
@@ -31,13 +32,48 @@ class HolochainServiceClient(
         }
       }
 
+  /// Entire process to setup an app
+  ///
+  /// Connect to service, install app, enable app, ensure app websocket
+  suspend fun setupApp(
```

Comment:
The actual logic to setup an app, moved from JS to here.

Reply:
Had a thought that this logic should probably be moved into the rust crate, so this library can remain a dumb wrapper. I'll address in a follow up: https://github.com/holochain/android-service-runtime/issues/80